### ISSUE TITLE
Add description to each page

### ISSUE THREAD

**Issue by bjohansebas** [expressjs/expressjs.com#1639]
This is a reminder to add the respective description to each page, which will improve the SEO of the page. This is a minor change and of low priority. If anyone wants to help, PRs are welcome.

**Comment by rehmansheikh222** [comment]
Hi, I am interested in resolving this issue. I would like to know if anyone else is already working on it?

**Comment by bjohansebas** [comment]
Hi @rehmansheikh222, no one has mentioned wanting to work on this, so I believe no one is working on this issue yet.

I am assigning this to you, If you have any questions, feel free to ask. Someone from the team will help you (as soon as we can).

**Comment by rehmansheikh222** [comment]
Hi @bjohansebas , thanks for letting me know. I am already working on this. Just I have some questions that I would like to clarify, does adding description to each page means every HTML file and by description, do you mean meta tags or something else.

**Comment by bjohansebas** [comment]
Hi @rehmansheikh222, I mean adding it to each markdown file where the content is located. For example, on this page https://github.com/expressjs/expressjs.com/blob/gh-pages/en/starter/examples.md?plain=1#L1, you can find the following data:
```markdown
---
layout: page
title: Express examples
menu: starter
lang: en
redirect_from: "/starter/examples.html"
---

...
```
It’s about adding the following option and providing an appropriate description for the page

```markdown
---
layout: page
title: Express examples
menu: starter
lang: en
description: "A description"
redirect_from: "/starter/examples.html"
---

...
```

The blog posts already contain it in case you want to use them as a reference: 
ref:
- https://github.com/expressjs/expressjs.com/blob/gh-pages/_posts/2024-09-29-security-releases.md
- https://github.com/expressjs/expressjs.com/blob/gh-pages/_posts/2024-10-15-v5-release.md

**Comment by rehmansheikh222** [comment]
Okay got it, thanks

**Comment by bjohansebas** [comment]
fixed by #1672
